simpl/config.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, *args, **kwargs):
         self._mutexgroup = None
 
     def __copy__(self):
+        """TODO(zns): add docstring."""
```

Comment:
Yes. This was just for PEP-8. Any idea how to disable them from a config side? pyintrc can have the setting below, but I don't know how to do that for flake8:
```
# Don't require docstrings on tests or Python Special Methods
no-docstring-rgx=(__.*__|[tT]est.*|setUp|tearDown)
```